### PR TITLE
visual an email indicator on home page 

### PR DESCRIPTION
**Pull request by heron2014**
Related: [issue 69](https://github.com/FAC-GM/app/issues/69)

maria_email

@nelsonic  can you please check this PR?

### CODE REVIEW

File: assets/css/enhanced.css [hunk 1]
```diff
@@ -59,6 +55,9 @@
   .favourite-form button {
     font-size: 1.5em;
   }
+   .fa-at {
+    padding-top: 20px;
```
Comment (iteles) [inline assets/css/enhanced.css]: Hard-coding a pixel value like this is unlikely to work nicely on a wide range of mobile devices. I would certainly recommend investigating other ways of positioning this!

**Comment by iteles** [comment]
@heron2014 Could we start using mobile device screenshots (maybe 1 iPhone 4, 1 iPhone 6 and an Android device of your choice) to make sure this is working properly on mobile too in terms of UI? :blush:

File: assets/css/enhanced.css (hunk 1 above)
Comment (heron2014) [inline assets/css/enhanced.css]: @iteles padding-top is only added to desktop version. On mobile this rule is omitted. I need to work soon on [issue 99](https://github.com/FAC-GM/app/issues/99) where px will be nicely converted to em :blush:

**Comment by heron2014** [comment]
@iteles we surely can :smile:

**Comment by nelsonic** [comment]
Agree with @iteles that we should avoid hard-coding pixel values.
This will be addressed in #99 so merging now.

**Pull request merged**